### DISCUSSION TITLE
[QUESTION] Why is the data tested by kbench very different from the official data?

### DISCUSSION THREAD

**Discussion by yyzhou0811** [longhorn/longhorn#8471] [General]
## Question
Hello, why is there a big gap between the longhorn reading and writing performance I measured and the official one?

## Environment
● Cloud provider: AWS
● Nodes: 4 (A node is both a master and a node)
● Storage: 4 NVMe SSD

Benchmarking Tools
● Use [kbench](https://github.com/yasker/kbench) as the benchmarking tool.

v1.6.1
● Machine: Linux/UNIX/c5d.2xlarge.x86
● OS: Amazon Linux 2
● Network throughput between nodes : 10Gbps

## Additional context

The following is the data I measured

RWO Volume

<img width="1317" alt="image" src="https://github.com/longhorn/longhorn/assets/30650510/861f77f6-fa3e-443f-a63b-521677ba1ed2">

Here are the official data
<img width="1034" alt="image" src="https://github.com/longhorn/longhorn/assets/30650510/1c1f2b6c-8d2d-4379-9b0c-26805bb9c37b">

**Comment by derekbit** [comment]
Can you provide more information such as number of replicas?
In addition, the official data is measured on baremetal machines.

**Reply by yyzhou0811** [reply]
My replicas are 3,The environment is as follows
Cloud provider: AWS
Nodes: 4 (A node is both a master and a node)
Storage: 4 NVMe SSD

Machine: Linux/UNIX/c5d.2xlarge.x86
OS: Amazon Linux 2
Network throughput between nodes : 10Gbps

**Reply by derekbit** [reply]
Got it. The difference is due to the platform.
cc @PhanLe1010
